sidechain/consensus/common/src/block_import.rs

```diff
@@ -77,6 +77,9 @@ where
 	/// Cleanup task after import is done.
 	fn cleanup(&self, signed_sidechain_block: &SB) -> Result<(), Error>;
 
+	/// Handles the cases where the sidechain block import failed.
+	fn handle_import_error(&self, signed_sidechain_block: &SB, error: Error) -> Result<(), Error>;
+
```

Comment:
That's how I imagined it 👍

Reply:
Yees, I'm finally getting closer to how the experts think 😊